### PR TITLE
Added most type signatures

### PR DESCRIPTION
**Pull request by Doomki**
Still to be done: Not completely sure what everything in library.py does, will look into it soon

### CODE REVIEW

**Comment by RitwikGupta** [comment]
Any reason why you closed this PR?

**Comment by Doomki** [comment]
Travis says some tests are failing, but I'm not able to run the tests on my laptop without getting a stack overflow. Should I just resubmit the pull request? Otherwise I will look into it tomorrow.

**Comment by RitwikGupta** [comment]
The build system currently has some bad tests, we have a PR working on that! But yes, please resubmit the PR so I can review and merge it.

**Comment by Doomki** [comment]
Ok, I reopened it. Mypy doesn't give errors for any files, so the signatures are probably ok.

File: PittAPI/dining.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def get_locations():
     return get_locations_by_status()
 
 
-def get_locations_by_status(status: str=None) -> List[Dict[str,str]]:
+def get_locations_by_status(status: str=None) -> List[Dict[str,Any]]:
```
Comment (azharichenko) [inline PittAPI/dining.py]: Don't forget to import `Any`, cause it's causing a `NameError` in the test log.

File: PittAPI/textbook.py [hunk 2]
```diff
@@ -1,7 +1,30 @@
 import grequests
 import requests
 
+from bs4 import BeautifulSoup
+from requests.exceptions import ConnectionError as RequestsConnectionError
+from typing import List, Dict, Any, Callable, Generator, Tuple
+
 BASE_URL = 'http://pitt.verbacompare.com/'
+
```
Comment (azharichenko) [inline PittAPI/textbook.py]: The `_fetch_term_codes` method was recently removed in pull request #73 .
Please remove the method, the `TERMS` constant, and the `requests.exception` import.

File: PittAPI/shuttle.py [hunk 3]
```diff
@@ -18,32 +18,33 @@
...
+def get_vehicle_route_stop_estimates(vehicle_id: str, quantity: int = 2) -> Dict[str,Any]:
     """Return {quantity} stop estimates for all active vehicles."""
-    payload = {"vehicleIdStrings": vehicle_id, "quantity": quantity}
+    payload = {"vehicleIdStrings": vehicle_id, "quantity": str(quantity)}
     response = sess.get("http://www.pittshuttle.com/Services/JSONPRelay.svc/GetVehicleRouteStopEstimates", params=payload)
     return response.json()
 
 
-def get_routes(api_key="8882812681"):
+def get_routes(api_key: str ="8882812681") -> Dict[str,Any]:
```
Comment (azharichenko) [inline PittAPI/shuttle.py]: Again, don't forget to import `Any`, it's giving me a `NameError` for this is the test log.

**Pull request merged**